Analysis/source/bbmuMuAnalyzer.cxx

```diff
@@ -90,16 +91,16 @@ class bbmumuAnalyzer : public BaseEventAnalyzer {
         sub_category.SetCutResult(SelectionCut::medPtNLO, event.GetHiggsTTMomentum(false).Pt() > 40 &&
                                                         event.GetHiggsTTMomentum(false).Pt() <= 100);
         sub_category.SetCutResult(SelectionCut::highPtNLO, event.GetHiggsTTMomentum(false).Pt() > 100);
-        sub_category.SetCutResult(SelectionCut::vlowPtLO, event.GetHiggsTTMomentum(false).Pt() <= 10);
-        sub_category.SetCutResult(SelectionCut::lowPtLO, event.GetHiggsTTMomentum(false).Pt() > 10 &&
-                                                        event.GetHiggsTTMomentum(false).Pt() <= 30);
-        sub_category.SetCutResult(SelectionCut::medPt1LO, event.GetHiggsTTMomentum(false).Pt() > 30 &&
-                                                        event.GetHiggsTTMomentum(false).Pt() <= 50);
-        sub_category.SetCutResult(SelectionCut::medPt2LO, event.GetHiggsTTMomentum(false).Pt() > 50 &&
-                                                        event.GetHiggsTTMomentum(false).Pt() <= 100);
-        sub_category.SetCutResult(SelectionCut::highPtLO, event.GetHiggsTTMomentum(false).Pt() > 100 &&
-                                                        event.GetHiggsTTMomentum(false).Pt() <= 200);
-        sub_category.SetCutResult(SelectionCut::vhighPtLO, event.GetHiggsTTMomentum(false).Pt() > 200);
+        sub_category.SetCutResult(SelectionCut::vlowPtLO, event.GetHiggsTTMomentum(false).Pt() <= ana_setup.pt_sel_bins.at(0));
```

Comment:
Here and below, add protection agianst smaller vector size:
`sub_category.SetCutResult(SelectionCut::vlowPtLO, ana_setup.pt_sel_bins.size() > 0 && event.GetHiggsTTMomentum(false).Pt() <= ana_setup.pt_sel_bins.at(0));`